### ISSUE TITLE
Fixes from AbilityNet Report - Tooltip content cannot be hovered over

### ISSUE THREAD

**Issue by min2ha** [ukwa/ukwa-ui#281]
Issue (WCAG-022) – Tooltip content cannot be hovered over [Ref: 1.4.13] Content on Hover or Focus New Issue – 21/05/2020

**Comment by min2ha** [comment]
Possible solution: Dismiss the tooltip on Escape button press without moving focus.
